pipeline/common/common.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+import logging
+
+import apache_beam as beam
+
+
+class Log(beam.DoFn):
```

Comment:
Would probably be better a `PTransform` because `| Log()` is easier to read and write than `| beam.ParDo(Log)`.

Reply:
Good point. Let's change that! But in another commit.